src/screens/OtherProfile/index.js

```diff
@@ -897,6 +897,19 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     alignItems: 'center'
   },
+  toggleSwitchContainer: {display: 'flex', alignSelf: 'flex-end', paddingVertical: 10},
+  bioAndSendChatContainer: (dynamicColors) => ({
+    backgroundColor: dynamicColors.primary,
+    borderRadius: 15,
+    paddingHorizontal: 10,
+    paddingTop: 10
+  }),
+  bioText: (dynamicColors) => ({
+    color: dynamicColors.text,
+    fontSize: 14,
+    fontWeight: '600',
+    lineHeight: 22
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here introduce dynamic colors to the styles, which is a good practice for supporting different themes (like dark mode and light mode) in the application. However, it's important to ensure that `dynamicColors` object contains the properties `primary` and `text`. If these properties are not guaranteed to exist in all cases, it could lead to runtime errors. Consider adding checks or fallbacks for these properties.

```diff
  bioAndSendChatContainer: (dynamicColors) => ({
    backgroundColor: dynamicColors.primary || 'defaultColor',
    borderRadius: 15,
    paddingHorizontal: 10,
    paddingTop: 10
  }),
  bioText: (dynamicColors) => ({
    color: dynamicColors.text || 'defaultTextColor',
    fontSize: 14,
    fontWeight: '600',
    lineHeight: 22
  }),
```
In this diff, `'defaultColor'` and `'defaultTextColor'` should be replaced with your actual default colors.


<!-- This is an auto-generated comment by OSS CodeRabbit -->